src/LaunchDarkly.Client/User.cs

```diff
@@ -175,5 +175,15 @@ public static User AndCustomAttribute(this User user, string attribute, List<str
 
             return user;
         }
+
+        public static User AndCustomAttribute(this User user, string attribute, List<int> value)
+        {
+            if (attribute == string.Empty)
+                throw new ArgumentException("Attribute Name can not be empty");
+
+            user.Custom.Add(attribute, new JArray(value.ToArray()));
```

Comment:
For some reason the array elements don't need to get cast to doubles (see test case)